docs/deployment/authorization-and-authentication/Spring-Security.md

```diff
@@ -0,0 +1,4 @@
+# Spring Security
+The cBioPortal app leverages [Spring Security](https://spring.io/projects/spring-security) to configure authorization and authentication. Some example settings can be found in the [security.properties.EXAMPLE](https://github.com/cBioPortal/cbioportal/blob/master/src/main/resources/security.properties.EXAMPLE) file. For more information on how to configure Spring Security, see the documentation:
+
+- [OAuth2 Configuration](https://docs.spring.io/spring-boot/reference/web/spring-security.html#web.security.oauth2)
```

Comment:
This could fit in the security props reference, but feels a little odd that in the `authorization-and-authentication` section of the docs there is no reference to Spring Security. This could maybe be a longer more general description of spring security, whereas the reference would be more about individual props